<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maintenance</class>
 <widget class="QWidget" name="Maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="MaintenanceWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 217, 159);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="AddSouvenirWidget">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="AddCollegeNamelabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>College Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="AddCollegeNameComboBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="AddSouvenirNamelabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>Souvenir Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AddSouvenirNameLine">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>Sourvenir Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="AddSouvenirPriceLabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>Price(USD):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="AddPriceSpinBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="AddSouvenirButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 167, 124);</string>
        </property>
        <property name="text">
         <string>Add Souvenir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AddCancelButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 167, 124);</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="AddSouvenirlabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>521</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 167, 124);</string>
     </property>
     <property name="text">
      <string>Add A New Souvenir</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="EditSouvenirWidget">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>621</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="EditSouvenirNamelabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>Souvenir Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditSouvenirNameLine">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>Sourvenir Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="EditSouvenirPriceLabel">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
          <property name="text">
           <string>Price(USD):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="EditPriceSpinBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 167, 124);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="EditSouvenirButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 167, 124);</string>
        </property>
        <property name="text">
         <string>Update Souvenir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EditCancelButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 167, 124);</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>581</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="EditSouvenirelabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 167, 124);</string>
        </property>
        <property name="text">
         <string>Change Souvenir Infomation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="EditCollegeName">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(83, 167, 124);</string>
        </property>
        <property name="text">
         <string>College Name:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
